src/domain/collaboration/post/post.entity.ts

```diff
@@ -2,14 +2,15 @@ import { Column, Entity, JoinColumn, OneToOne } from 'typeorm';
 import { IPost } from './post.interface';
 import { NameableEntity } from '@domain/common/entity/nameable-entity/nameable.entity';
 import { Room } from '@domain/communication/room/room.entity';
+import { UUID_LENGTH } from '@common/constants';
 
 @Entity()
 export class Post extends NameableEntity implements IPost {
   @Column('text')
   type!: string;
 
-  @Column('char', { length: 36, nullable: true })
-  createdBy!: string;
+  @Column('char', { length: UUID_LENGTH, nullable: true })
+  createdBy?: string;
```

Comment:
I have left `createdBy` to be optional, since the code creates a lot of things and we don't have a standartized "creator" in these cases.
Furthermore it would be great to be a FK instead of a string value - that would only be possible of course, if the ID is coming from a single table, i.e. `user.

Reply:
To consider a FK, but it cannot in many cases be a TypeORM relation  as that leads to all kinds of circular dependencies. For later.